kombu/transport/redis.py

```diff
@@ -1077,11 +1075,14 @@ def _sentinel_managed_pool(self, async=False):
 
         additional_params.pop('host', None)
         additional_params.pop('port', None)
-
+        connection_list = list()
+        for url in self.connection.client.alt:
```

Comment:
what does .clinet.alt do?

Reply:
It contains the alternate endpoints. It is populated over here: https://github.com/celery/kombu/blob/master/kombu/connection.py#L165-L189

It goes through that logic in the unit test provided in this PR.